pants_plugins/sendwave/pants_docker/subsystem.py

```diff
@@ -24,6 +24,15 @@ class Docker(Subsystem):
         help="If true: the plugin will report output of `docker build`",
     )
 
+    cache_from = StrOption(
+        "--cache-from",
+        default=None,
+        help=(
+            "Specify a registry that will be used for caching via the "
+            "docker build --cache-from option (requires BuildKit to be installed)"
+        ),
+    )
+
```

Comment:
I think this wants to be on the target - not the subsystem, cause this can only be set once per `pants` invocation, so when we build multiple images it will be the same for each one (i.e. multiple targets are listed in the targets.txt file!)

Reply:
Hmm good point! I'll move this to the target 👍🏻